src/test/resources/fixtures/single.yaml

```diff
@@ -1,8 +1,4 @@
 ---
-serial-number: 1
-hash: "someHash1"
-entry:
-  fields:
-  - "field1"
-  register: "value1"
-  text: "The Entry 1"
+entry-number: "1"
```

Comment:
ah okay, and https://openregister.github.io/specification/#json-representation says

> All field values MUST be encoded as JSON strings.

for the JSON representation. We should probably decide if we want to do the same for YAML or not.